docs/localization-overview.md

```diff
@@ -0,0 +1,30 @@
+# Localization Overview
+
+You can localize your Catalyst storefront so that it appears in the shopper's preferred culture or language throughout their browsing and checkout experience. 
+
+> [!NOTE] 
+> Internationalization support in Catalyst is a work in progress. Full multilingual support in headless channels, like Catalyst, will be added in future releases.
+
+## Use cases
+
+Catalyst supports the following scenarios:
+
+1. You are selling internationally and would like to tailor your storefront to multiple countries. Each country has its own language, currency, payment method, etc.
+
+2. You are selling within a country would like to accommodate multiple languages within that country
+ 
+To fully localize a store for a language or region, you will need to customize product catalog and storefront content in the BigCommerce control panel.
+
+## Locale-specific URLs
+
+Catalyst supports the following locale-specific URLs that you can use to localize your storefronts. Learn more about the pro and cons for each [locale-specific URL](https://developers.google.com/search/docs/specialty/international/managing-multi-regional-sites#locale-specific-urls) in the Google Documentation.
+
+Currently, each Catalyst storefront can only support a single language. To display multiple languages, we recommend setting up a separate channel for each language.
+
+| URL structure | Example | Recommended use case | How to implement for Catalyst | 
```

Comment:
@bookernath Not sure if I should have the "Recommended use case" column until after we support translated language content within the same channel? To my understanding, until then, all URL structures point to different channels (all URL structures could work equally well for Multi-GEO or Multi-Lang)

Reply:
I think this is fine for now and we can iterate on it later.